client/templates/initiatives/mine/mine.html

```diff
@@ -14,7 +14,7 @@
       {{/each}}
     </div>
     <div class="initiative-mine row center-align">
-        <a class="button btn" href="{{pathFor 'initiatives.create'}}">New Initiative</a>
+        <a class="button waves-effect waves-teal btn-flat" href="{{pathFor 'initiatives.create'}}">New Initiative</a>
```

Comment:
Would you mind trying the following classes here to match the create initiative button on the create screen?

    class="btn waves-effect waves-light blue darken-1"

or something similar perhaps?